device_services/meta/parameters.md

```diff
@@ -167,4 +167,16 @@ Parameters service allows for advanced configuration of a device.
   "uid": "961d2024-0d85-43ae-82e8-dd93085aaf78",
   "topic": "pt:j1/mt:evt/rt:dev/rn:zw/ad:1/sv:parameters/ad:149_0"
 }
-```
\ No newline at end of file
+```
+
+## Predefined parameters
+
+Parameters service is primarily intended to support vendor-specific configuration parameters that can vary for every device.
+However, there are some predefined parameters which have a standardized meaning and are expected to have uniform implementation across devices which support them.
+
+| Parameter ID          | Name                | Value Type | Requirements                              | Description                                                                                                                                              | 
+|-----------------------|---------------------|------------|-------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `heat_controll_mode`  | Heat control mode   | `string`   | `thermostat` or `power_regulator` service | Allows to switching heating control mode between thermostat and power regulator. Accepted values are `reg` for power regulator and `tht` for thermostat. |
```

Comment:
```suggestion
| `heat_control_mode`  | Heat control mode   | `string`   | `thermostat` or `power_regulator` service | Allows to switching heating control mode between thermostat and power regulator. Accepted values are `reg` for power regulator and `tht` for thermostat. |
```

Reply:
Maybe it will be worthy to add in description, that after mode switching, device should resend new inclusion report?

Reply:
Yep, done.